frontend/src/views/c-zoom.vue

```diff
@@ -5,12 +5,12 @@
   .toolbar--multiline(v-if="filteredUser.commits.length && totalCommitMessageBodyCount")
     a(
       v-if="expandedCommitMessagesCount < totalCommitMessageBodyCount",
-      v-on:click="toggleAllCommitMessagesBody(true)"
-    ) show all commit messages
+      v-on:click="toggleAllCommitMessagesBody(true); toggleDiffstatView(true);"
+    ) show all commit details
     a(
       v-if="expandedCommitMessagesCount > 0",
-      v-on:click="toggleAllCommitMessagesBody(false)"
-    ) hide all commit messages
+      v-on:click="toggleAllCommitMessagesBody(false); toggleDiffstatView(false);"
+    ) hide all commit details
```

Comment:
I feel that the individual toggling of the commit message body should not also toggle the visibility of the diffstat. This is because the purpose of the diffstat is to provide the user with a "bird's eye view" of the changes in a commit in relation to one another. Users already zooming in on a particular commit can refer to the lines changed.

Therefore, toggling each of the diffstats individually might not be necessary, save for the (rare) scenarios where diffstats occupying many lines may need to be hidden to reduce clutter.

In these cases, perhaps it would be better to add a separate individual toggle button for showing/hiding the diffstat, so that users can still tell, based on the presence of the existing toggle button, which commits have message bodies.